traitsui/tests/editors/test_range_editor.py

```diff
@@ -68,30 +69,75 @@ def check_set_with_text_valid(self, mode):
             self.assertEqual(model.value, 1)
             number_field = tester.find_by_name(ui, "value")
             text = number_field.locate(locator.WidgetType.textbox)
-            if is_windows and is_wx() and mode == 'text':
-                # For RangeTextEditor on wx and windows, the textbox
-                # automatically gets focus and the full content is selected.
-                # Insertion point is moved to keep the test consistent
-                text.target.textbox.SetInsertionPointEnd()
             text.perform(command.KeyClick("0"))
             text.perform(command.KeyClick("Enter"))
             displayed = text.inspect(query.DisplayedText())
             self.assertEqual(model.value, 10)
             self.assertEqual(displayed, str(model.value))
 
     def test_simple_slider_editor_set_with_text_valid(self):
-        return self.check_set_with_text_valid(mode='slider')
+        return self.check_slider_set_with_text_valid(mode='slider')
 
     def test_large_range_slider_editor_set_with_text_valid(self):
-        return self.check_set_with_text_valid(mode='xslider')
+        return self.check_slider_set_with_text_valid(mode='xslider')
 
     def test_log_range_slider_editor_set_with_text_valid(self):
-        return self.check_set_with_text_valid(mode='logslider')
+        return self.check_slider_set_with_text_valid(mode='logslider')
 
     def test_range_text_editor_set_with_text_valid(self):
-        return self.check_set_with_text_valid(mode='text')
+        model = RangeModel()
+        view = View(
+            Item(
+                "value",
+                editor=RangeEditor(low=1, high=12, mode="text")
+            )
+        )
+        tester = UITester()
+        with tester.create_ui(model, dict(view=view)) as ui:
+            # sanity check
+            self.assertEqual(model.value, 1)
+            number_field_text = tester.find_by_name(ui, "value")
+            if is_windows and is_wx():
+                # For RangeTextEditor on wx and windows, the textbox
+                # automatically gets focus and the full content is selected.
+                # Insertion point is moved to keep the test consistent
+                number_field_text.target.textbox.SetInsertionPointEnd()
+            number_field_text.perform(command.KeyClick("0"))
+            number_field_text.perform(command.KeyClick("Enter"))
+            displayed = number_field_text.inspect(query.DisplayedText())
+            self.assertEqual(model.value, 10)
+            self.assertEqual(displayed, str(model.value))
+
+    # SimpleSpinEditor not yet implemented on Wx.
+    @requires_toolkit([ToolkitName.qt])
+    def test_simple_spin_editor_set_with_text_valid(self):
+        from traitsui.testing.tester.registry import TargetRegistry
+        from traitsui.testing.tester.qt4.implementation.range_editor import (
+            _register_simple_spin
```

Comment:
Since this is only used by this module, could we just define this function locally in this module?
The preceding underscore is used to signal when something is private. It being imported from outside goes against this convention.

If we will elevate the implementation to the default registry and to use it for RangeEditor tests, this function will be discovered here and be moved. Much like the `set_combobox_index`, `finish_combobox_text_entry` etc removed from test_enum_editor when the tester implementation was added.